### PR TITLE
testbench: Fixed / updated travis test script

### PR DESCRIPTION
**Pull request by alorbach**
- Fixed issue in relpsess found using clang (Possible dupl of NULL
  string).
- Fixed issue found in gcc build in receive.c (variable initialization).
- Exclude generated config.h from codestyle check

### CODE REVIEW

File: devtools/default_dev_container [hunk 1]
```diff
@@ -1 +1 @@
-rsyslog/rsyslog_dev_base_ubuntu:16.04
+rsyslog/rsyslog_dev_base_ubuntu:19.10
```
Comment (rgerhards) [inline devtools/default_dev_container:1]: I would be a bit careful here - 19.10 is an interim release with EOL in around 3 months. Isn't 18.04 sufficient? Else I can see that we get 20.04 in place.

Comment (alorbach) [reply devtools/default_dev_container:1]: @rgerhards just got it from rsyslog 
https://github.com/rsyslog/rsyslog/blob/master/devtools/default_dev_container

Comment (rgerhards) [reply devtools/default_dev_container:1]: oops ;-) some more thing to do... OK, I'll look at it.

Comment (alorbach) [reply devtools/default_dev_container:1]: So we leave it for 19:10 for now? At least we can use latest clang here which already found a new race while making this PR here.

**Pull request merged**